src/main/java/com/netgrif/application/engine/integration/plugins/config/PluginProperties.java

```diff
@@ -0,0 +1,58 @@
+package com.netgrif.application.engine.integration.plugins.config;
```

Comment:
This may be an old implementation of plugins. It is now solved through gRPC and with PF4J. These classes should be reviewed.